docs/fastdds/dds_layer/topic/contentFilteredTopic/defaultFilter.rst

```diff
@@ -0,0 +1,362 @@
+
+          .. code:: IDL
+
+              struct Color
+              {
+                  octet red;
+                  octet green;
+                  octet blue;
+              };
+
+              struct Shape
+              {
+                  long points[4];
+                  Color color;
+              };
+
+- **INTEGERVALUE**: Any series of digits, optionally preceded by a plus or minus sign, representing a decimal integer
+  value within the range of the system.
+  A hexadecimal number is preceded by ``0x`` and must be a valid hexadecimal expression.
+
+  .. productionlist::
+      INTEGERVALUE: (["+","-"])? Integer
+      Integer: (["0"-"9"])+ | ["0x","0X"](["0"-"9", "A"-"F", "a"-"f"])+
+
+  An example of INTEGERVALUE:
+
+  .. code::
+
+      value = -10
+
+- **CHARVALUE**: A single character enclosed between single quotes.
+
+  .. productionlist::
+      CHARVALUE: "'" Character "'"
+      Character: ~["\n"]
+
+  An example of CHARVALUE:
+
+  .. code::
+
+      value = 'c'
+
+- **FLOATVALUE**: Any series of digits, optionally preceded by a plus or minus sign and optionally including a floating
+  point (``.``).
+  A power-of-ten expression may be postfixed, which has the syntax e:sup:`n`, where ``n`` is a number, optionally
+  preceded by a plus or minus sign.
+
+  .. productionlist::
+      FLOATVALUE: (["+"], "-"])? (Integer Exponent | Integer Fractional | Integer Fractional Exponent)
+      Fractional: "." Integer
+      Exponent: ["e","E"] (["+"], "-"])? Integer
+
+  An example of FLOATVALUE:
+
+  .. code::
+
+      value = 10.1e-10
+
+- **STRINGVALUE**: Any series of characters encapsulated in single quotes, except a new-line character or a right quote.
+  A string starts with a left or right quote, but ends with a right quote.
+
+  .. productionlist::
+      STRINGVALUE: ["'"] ~["'", "\r", "\n"] ["'"]
+
+  An example of STRINGVALUE:
+
+  .. code::
+
+      value = 'This is a string'
+
+
+- **ENUMERATEDVALUE**: An enumerated value is a reference to a value declared within an enumeration.
+  Enumerated values consist of the name of the enumeration label enclosed in single quotes.
+  The name used for the enumeration label must correspond to the label names specified in the IDL definition of the
+  enumeration.
+
+  .. productionlist::
+      ENUMERATEDVALUE: ["'"] ~["'", "\r", "\n"] ["'"]
+
+  An example of ENUMERATEDVALUE:
+
+  .. tabs::
+
+      .. tab:: Filter expression
+
+          .. code::
+
+              value = 'ENUM_VALUE_1'
+
+      .. tab:: Associated IDL
+
+          .. code::
+
+              enum MyEnum
+              {
+                  ENUM_VALUE_1,
+                  ENUM_VALUE_2,
+                  ENUM_VALUE_3
+              };
+
+              struct Enumerators
+              {
+                  MyEnum value;
+              };
+
+- **BOOLEANVALUE**: Can either be `true` of `false`, case sensitive.
+
+  .. productionlist::
+      BOOLEANVALUE: ["TRUE", "true", "FALSE", "false"]
```

Comment:
I suggest following the same order as in the grammar before. So `BOOLEANVALUE` should be before `INTEGERVALUE`.